k2/csrc/cuda/types.h

```diff
@@ -0,0 +1,268 @@
+// k2/csrc/cuda/types.h
```

Comment:
I thought we (Dan or I) have deleted types.h and ragged_shape.cc before, seems you forget to sync

Reply:
oh, let me sync it.